manifests/gcp_marketplace/schema.yaml

```diff
@@ -101,33 +101,59 @@ properties:
     x-google-marketplace:
       type: NAMESPACE
   serviceAccountCredential:
-    title: Service Account credentials used to call other GCP services.
+    title: |-
+      Service Account credentials used to call other GCP services.
```

Comment:
wonding whether we call it "GCP Service Account credentials used ...". 
As talked last time, we have two types of service account, one is K8s and other is GCP.

Reply:
good point. updated.